packages/web-components/src/text-field/text-field.styles.ts

```diff
@@ -12,6 +12,38 @@ import {
   neutralOutlineHoverBehavior,
   neutralOutlineRestBehavior,
 } from '../styles';
+import { appearanceBehavior } from '../utilities/behaviors';
+
+export const TextFieldFilledStyles = css`
+  :host(.filled) .root {
+    background: ${neutralFillRestBehavior.var};
+    border-color: transparent;
+  }
+
+  :host(.filled:hover:not(.disabled)) .root {
+    background: ${neutralFillHoverBehavior.var};
+    border-color: transparent;
+  }
+`.withBehaviors(
+  neutralFillHoverBehavior,
+  neutralFillRestBehavior,
+  forcedColorsStylesheetBehavior(
+    css`
+      :host(.filled) .root {
```

Comment:
@khamudom while we're here - can you update all instances of these so they target the appearance attribute?

Reply:
updated